### PR TITLE
Ghostery 10: Firefox scriptlet injection

### PR DESCRIPTION
**Pull request by chrmod**
Injecting blobs allow scriptlets to by pass CSP. This technique was already in use in Ghostery 8 and with autoconsent.

### CODE REVIEW

File: extension-manifest-v3/src/background/adblocker.js [hunk 1]
```diff
@@ -232,18 +232,34 @@ ${scripts.join('\n\n')}}
...
-        {
-          createScript: (s) => s,
-        },
-      );
-      content = trustedTypePolicy.createScript(content);
+    script.async = false;
+
+    let scriptlets = decodeURIComponent(code);
+
+    if (navigator.userAgent.includes('Firefox')) {
```
Comment (smalluban) [inline extension-manifest-v3/src/background/adblocker.js:240]: I think this is place to use `__PLATFORM__`

```suggestion
    if (__PLATFORM__ === 'firefox') {
```

**Review by smalluban — CHANGES_REQUESTED** [review]

**Comment by smalluban** [comment]
This is critical path for extension. Please test it before merging @GRadziejewski

**Review by smalluban — COMMENTED** [review]
LGTM.

**Review by smalluban — APPROVED** [review]

**Comment by chrmod** [comment]
It looks like blob injection is not needed when `chrome.scripting.executeScript` is in use.

**Pull request closed without merging**